src/main/resources/reference.conf

```diff
@@ -20,20 +20,30 @@ jmt {
 
   transceiver {
       recv {
-          # the size of the queue that handles incoming rtp/rtcp packets. Each
-          # local endpoint/rtp receiver has such a queue and it temporarily stores the
-          # packets that the endpoint is sending, right after an srtp packet has
-          # been received.
-          queue-size=1024
+        # The size of the queue that handles incoming rtp/rtcp packets. This is
+        # a per-participant queue and it's where we temporarily store the
+        # packets that the endpoint is sending, right after an srtp packet has
+        # been received.
+        #
+        # Assuming 300pps for high-definition, 200pps for standard-definition
+        # and 150pps for low-definition, this queue is fed 650pps, so its size
+        # in terms of millis is 650/1024*1000 ~= 1260ms.
```

Comment:
Where do the numbers for pps come from? The assumption for 150pps for the def seems very high, but you're not counting audio.

Your quotients in the time estimation are flipped: with the size bumped to 2048 it should be 2048/650 = 3.15s

Why the bump to 2048?

Reply:
> Where do the numbers for pps come from? The assumption for 150pps for low def seems very high, but you're not counting audio.

They're from memory, and it's over-estimated. Do you have more accurate numbers to use?

> Your quotients in the time estimation are flipped: 

:face_palm:

> Why the bump to 2048?

I'll re-run the calculations and provide a better default.

Reply:
@bgrozev done